plugins/ci/skills/slack-pr-notify/slack-pr-notify.sh

```
@@ -0,0 +1,157 @@
+
+  _MENTIONS="${_MENTIONS# }"
+}
+
+# _post_webhook <json_payload>
+#   Posts to SLACK_WEBHOOK_URL, suppressing set -x to avoid leaking the URL.
+#   Never fatal — returns 0 regardless of outcome.
+_post_webhook() {
+  local payload="$1"
+
+  # Save and disable tracing to protect the webhook URL.
+  [[ $- == *x* ]] && local was_tracing=true || local was_tracing=false
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)